agent/lm_agent/backend_utils.py

```diff
@@ -48,6 +48,44 @@ class Config:
     grace_time: int
 
 
+class BackendBookingRow(BaseModel):
+    """
+    NOTE: This is a copy of the schema from the backend.
+          If the schema changes upstream in a non-reverse-compatible
+          way, this schema should cause errors in deserialization.
+    """
+
+    class Config:
+        extra = "ignore"
+
+    job_id: str
+    product_feature: str
+    booked: int
+    config_id: int
+    lead_host: str
+    user_name: str
+    cluster_name: str
+
+
+async def get_bookings_from_backend(cluster_name: str = None) -> List[BackendBookingRow]:
```

Comment:
Seems like this would cause a Mypy error. Probably need to add Optional:

```suggestion
async def get_bookings_from_backend(cluster_name: Optional[str] = None) -> List[BackendBookingRow]:
```